dockerfiles/jupyter/Dockerfile

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM jupyter/scipy-notebook:python-3.8.8
+
+ARG MLRUN_PYTHON_VERSION=3.9.13
+
+FROM jupyter/scipy-notebook:python-${MLRUN_PYTHON_VERSION}
```

Comment:
Lets only depend on images which are managed in our registries, unfortunately we already experienced situations where we depended on third party hosted image registry which decided to delete the image. 

So just pull the image locally, tag with `quay.io/mlrun/jupyter-scipy-notebook` and push to our registry.